src/postamble.js

```diff
@@ -225,7 +225,7 @@ function run(args) {
   if (preloadStartTime === null) preloadStartTime = Date.now();
 
   if (runDependencies > 0) {
-#if ASSERTIONS
+#if ASSERTIONS && !BINARYEN_ASYNC_COMPILATION
```

Comment:
i think we don't need this addition. when assertions are on, this can be shown. they aren't on in production. and assertions is our setting for 'do more checks and logging at runtime' (although maybe we should have a separate setting for logging, but that's a side issue)